src/styles/blocks/card.scss

```
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+
+  display: flex;
+  flex-direction: column;
+
+  border-radius: 5px;
+  border: 1px solid $card-border;
+  background: $white;
+
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  &__image {
+    margin: 0 auto 36px;
+  
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px; /* 150% */
+    margin: 4px 0;
+  }
+
+  &__code {
+    color: $secondary;
+    font-size: 10px;
+    line-height: 14px; /* 140% */
+    margin: 0 0 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &--value {
+      margin: 0;
+      color: $main-accent;
+      text-align: right;
+      font-size: 10px;
+      line-height: 14px; /* 140% */
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 0;
+
+    &--text {
+      color: $secondary;
+      font-size: 12px;
+      line-height: 18px; /* 150% */
+    }
+
+    &--value {
+      color: $main-accent;
+      text-align: right;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px; /* 112.5% */
+    }
+  }
+
+  &__button {
+    margin-top: 0;
+  
+    height: 40px;
+    border-radius: 5px;
+    background: $blue-accent;
+  
+    color: $white;
+    text-align: center;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 40px; /* 114.286% */
+    text-transform: uppercase;
+
+    &:hover {
+      box-sizing: border-box;
+      border: 1px solid $blue-accent;
+      background: $white;
+      color: var(--blue-accent);
```

Why do you use css syntax here? 👀

---

to change button styles on hover

---

Will it work like this?
```suggestion
      color: $blue-accent;
```

---

yes https://chrishryts.github.io/layout_product-cards/

---

But you didn't change your code. You still have `color: var(--blue-accent)`. 

Please double-check my comments and your code.